src/items.rs

```diff
@@ -2,23 +2,45 @@ use chrono::DateTime;
 use chrono::NaiveDate;
 use chrono_tz::Tz;
 use colored::*;
+use inquire::Select;
 use serde::{Deserialize, Serialize};
 use std::cmp::Reverse;
+use std::fmt::Display;
 
 use crate::config::Config;
-use crate::{config, items, request, time};
+use crate::{items, request, time};
 
 #[derive(Serialize, Deserialize, Debug, Clone, Eq, PartialEq)]
 pub struct Item {
     pub id: String,
     pub content: String,
-    pub priority: u8,
+    pub priority: Priority,
     pub checked: bool,
     pub description: String,
     pub due: Option<DateInfo>,
     pub is_deleted: bool,
 }
 
+#[derive(serde_repr::Serialize_repr, serde_repr::Deserialize_repr, Debug, Clone, Eq, PartialEq)]
+#[repr(u8)]
+pub enum Priority {
+    Default = 1,
+    Low = 2,
+    Normal = 3,
+    High = 4,
+}
+
+impl Display for Priority {
+    fn fmt(&self, f: &mut std::fmt::Formatter<'_>) -> std::fmt::Result {
+        match self {
+            Priority::Default => write!(f, "DEFAULT"),
+            Priority::Low => write!(f, "LOW"),
+            Priority::Normal => write!(f, "NORMAL"),
```

Comment:
how about LOW/MEDIUM/HIGH just because MEDIUM is usually associated with the other two words.

Reply:
I have renamed it according to your suggestion.
Also I often use normal in context of task management, TMS and so on. But English is not my first language and I don't have a strong opinion about it =)